libibumad/umad.h

```diff
@@ -225,14 +224,14 @@ static inline void umad_free(void *umad)
 	free(umad);
 }
 
-#ifndef ntohll
-#undef htonll
-#undef ntohll
+#ifndef be32_to_cpul
+#undef cpu_to_be32l
+#undef be32_to_cpul
 /* Users should use the glibc functions directly, not these wrappers */
-static inline __attribute__((deprecated)) uint64_t htonll(uint64_t x) { return htobe64(x); }
-static inline __attribute__((deprecated)) uint64_t ntohll(uint64_t x) { return be64toh(x); }
-#define htonll htonll
-#define ntohll ntohll
+static inline __attribute__((deprecated)) uint64_t cpu_to_be32l(uint64_t x) { return cpu_to_be64(x); }
+static inline __attribute__((deprecated)) uint64_t be32_to_cpul(uint64_t x) { return be64_to_cpu(x); }
+#define cpu_to_be32l cpu_to_be32l
+#define be32_to_cpul be32_to_cpul
 #endif
```

Comment:
That was unintended. Should have been fixed in the latest version of this patch series.